core/src/main/java/org/apache/brooklyn/core/entity/AbstractEntity.java

```diff
@@ -1484,9 +1484,11 @@ public void unsubscribeAll() {
         }
         
         protected SubscriptionContext getSubscriptionContext() {
-            synchronized (AbstractEntity.this) {
-                return getManagementSupport().getSubscriptionContext();
-            }
```

Comment:
fix looks good, agree `EMS` handles synching is sufficient and means we should not try to do so here